jscomp/bsb/bsb_pkg.ml

```diff
@@ -46,6 +46,11 @@ let check_dir dir =
   | true -> Some(dir)
   | false -> None
 
+let is_same_paths a b =
+  if a = b
+  then true
+  else (Bsb_real_path.real_path a) = (Bsb_real_path.real_path b)
+
```

Comment:
can we push `is_same_paths` into `bsb_real_paths` so that only such function is used.
In general, you don't expect path comparison triggers io operation, can we make it more informative in the name, e.g, `is_same_paths_via_io` etc

Reply:
yeap will do